modules/consensus/state.go

```diff
@@ -128,6 +139,10 @@ func New(saveDir string) (*State, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	// spawn resynchronize loop
+	go s.threadedResynchronize()
```

Comment:
Are there tests which make sure that the resynchronize loop is working correctly? I know it wasn't before this PR.